plugins/reporters/spdx/src/main/kotlin/Extensions.kt

```diff
@@ -217,7 +217,9 @@ internal fun Package.toSpdxPackage(
                 .sorted()
         },
         name = id.name,
+        originator = authors.joinToString(prefix = SpdxConstants.PERSON).takeUnless { it == SpdxConstants.PERSON },
```

Comment:
How about `authors.takeIf { it.isNotEmpty() }?.let { it.joinToString(prefix = SpdxConstants.PERSON) }` (same below)?

Reply:
Done slightly differently.